experimenter/tests/tools/sdk_eval_check.py

```diff
@@ -12,8 +12,8 @@ def load_parser():
 
 
 def load_context():
-    if os.path.exists("app_context.json"):
-        with open("app_context.json") as file:
+    if Path.exists("app_context.json"):
```

Comment:
Path.exists is a member function. This won't work.

```
>>> from pathlib import Path
>>> Path.exists("s")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Python39\lib\pathlib.py", line 1424, in exists
    self.stat()
AttributeError: 'str' object has no attribute 'stat'
```

```suggestion
    if Path("app_context.json").exists():
```